src/bitmask.mojo

```diff
@@ -1,36 +1,53 @@
-from bit import pop_count
+from bit import pop_count, bit_not
 from filter import MaskFilter
 
 
 @value
 @register_passable
 struct BitMask(Stringable):
-    """BitMask is a 128 bit bitmask.
+    """BitMask is a 256 bit bitmask."""
 
-    It is also a [Filter] for including certain components.
-    Use [all] to create a mask for a list of component IDs.
-    A mask can be further specified using [Mask.without] or [Mask.exclusive].
-    """
-
-    alias word_size: Int = 64
-    alias total_bits: Int = 2 * Self.word_size
     alias IndexType = UInt8
-    var lo: UInt64  # First 64 bits of the mask
-    var hi: UInt64  # Second 64 bits of the mask
+    alias total_bits = 256
+    alias total_bytes = Self.total_bits // 8
+
+    var _bytes: SIMD[DType.uint8, Self.total_bytes]
+
+    @always_inline
+    fn __init__(inout self, bytes: SIMD[DType.uint8, Self.total_bytes]):
+        """Initializes the mask with the given bytes."""
+        self._bytes = bytes
 
+    @always_inline
+    fn __init__(inout self, bits: VariadicList[BitMask.IndexType]):
+        """Initializes the mask with the bits at the given indices set to True.
+        """
+        self._bytes = SIMD[DType.uint8, Self.total_bytes]()
+        for bit in bits:
```

Comment:
Is `bits` like a list of bools? Asking to clarify whether the loop is really required...

Reply:
Ah, I see, it is a list of indices...